frontend/src/lib/OutputArtifactLoader.ts

```diff
@@ -311,6 +311,37 @@ export class OutputArtifactLoader {
         return buildArtifactViewer(script);
       }),
     );
+    const EvaluatorArtifactUris = filterArtifactUrisByType(
+      'ModelEvaluation',
+      artifactTypes,
+      artifacts,
+    );
+    viewers = viewers.concat(
+      EvaluatorArtifactUris.map(uri => {
+        const configFilePath = uri + '/eval_config.json';
+        const script = [
+          `import json`,
+          `import tensorflow as tf`,
+          `import tensorflow_model_analysis as tfma`,
+          `from ipywidgets.embed import embed_minimal_html`,
+          `from IPython.core.display import display, HTML`,
+          `config_file=tf.io.gfile.GFile('${configFilePath}', 'r')`,
+          `config=json.loads(config_file.read())`,
+          `featureKeys=list(filter(lambda x: 'featureKeys' in x, config['evalConfig']['slicingSpecs']))`,
```

Comment:
please move these Python codes to visulization server. please.
You can simply introduce a new enum type for it, cost is not big.

Reply:
I don't feel this is necessary, but if cost is not high, we can do so too.
I'm not familiar with visualization server code, @rmgogogo can you provide a reference for @jingzhang36 ?

Reply:
I'll leave a TODO for this. If we decide to move, we'll move the statistics,  schema and the others together.

Reply:
please check the TFDV patch as a reference. The cost is not big. Just need to add a proto enum and a new python file in types/ folder. It would largely help you well format these python codes.

do prefer take a try on that path.

Reply:
TODO works so that we can catch beta.